### PR TITLE
Pull

### PR DESCRIPTION
**Pull request by Snow-Ye**

### CODE REVIEW

File: code/src/components/AddCarUseState.tsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
+// 'use client';
```
Comment (Alisa007) [inline code/src/components/AddCarUseState.tsx:1]: You should not commit component like this

File: code/tsconfig.json [hunk 2]
```diff
@@ -23,6 +23,12 @@
       "@/*": ["./src/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    ".next/types/**/*.ts",
+    "**/*.js"
```
Comment (Alisa007) [inline code/tsconfig.json:31]: Why did you add it but never used

File: code/src/public/line-4.svg
Comment (Alisa007) [inline code/src/public/line-4.svg:1]: Lines and rectangles should be done with css/html

File: code/src/public/add.svg
Comment (Alisa007) [inline code/src/public/add.svg:1]: I already suggested to use icons as components, easier to color and modify and better for performance and code looks cleaner

File: code/src/components/Sidebar.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+// Sidebar.tsx
```
Comment (Alisa007) [inline code/src/components/Sidebar.tsx:1]: There is not point to leave comments like that

File: code/src/components/RoundedInput.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import React, { ChangeEvent } from 'react';
+
+interface RoundedInputProps {
+  type: string;
+  placeholder: string;
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+}
+
+// Use function declaration and simplify the return statement
```
Comment (Alisa007) [inline code/src/components/RoundedInput.tsx:9]: Strange comment

File: code/src/components/EditCar.tsx [hunk 5]
```diff
@@ -0,0 +1,144 @@
+import React from 'react';
```
Comment (Alisa007) [inline code/src/components/EditCar.tsx:1]: You dont need to have react imported everywhere

File: code/src/components/AddCar.tsx [hunk 6]
```diff
@@ -0,0 +1,165 @@
+'use client';
```
Comment (Alisa007) [inline code/src/components/AddCar.tsx:1]: EditCar and AddCar should be the same component, they look identical in design

File: code/src/app/dashboard/carinformation/page.tsx [hunk 7]
```diff
@@ -1,9 +1,21 @@
 import DashboardLayout from '../../../components/DashboardLayout';
+import CarInfo from '../../../components/CarInfo';
 
 export default function page() {
   return (
     <DashboardLayout>
-      <p>car information</p>
+      <CarInfo
```
Comment (Alisa007) [inline code/src/app/dashboard/carinformation/page.tsx:7]: EditCar and CarInfo should be the same component, they look identical in design

**Review by Alisa007 — APPROVED** [review]

**Review by ncruz93 — APPROVED** [review]
